src/Core.UnitTests/Core.UnitTests.csproj

```diff
@@ -19,7 +19,7 @@
     <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
     <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
 
-    <PackageReference Include="Moq" Version="4.14.5" />
+    <PackageReference Include="Moq" Version="4.18.2" />
```

Comment:
We could perhaps move the common test packages to a shared file (maybe `SonarLint.Test.props`), but not as part of this change.